dataprofiler/tests/speed_tests/structured_throughput_testing.py

```diff
@@ -0,0 +1,114 @@
+import time
+import json
+import random
+from collections import defaultdict
+
+import numpy as np
+import tensorflow as tf
+
+try:
+    import sys
+    sys.path.insert(0, '..')
+    import dataprofiler as dp
+except ImportError:
+    import dataprofiler as dp
+    
+
+# suppress TF warnings
+tf.compat.v1.logging.set_verbosity(tf.compat.v1.logging.ERROR)
+
+
+################################################################################
+######################## set any optional changes here #########################
+################################################################################
+options = dp.ProfilerOptions()
+
+# these two options default to True if commented out
+options.structured_options.multiprocess.is_enabled = False
+# options.structured_options.data_labeler.is_enabled = False
+
+# parameter alteration
+ALLOW_SUBSAMPLING = True # profiler to subsample the dataset if large
+PERCENT_TO_NAN = 0.0  # Value must be between 0 and 100
+
+
+sample_sizes = [100, 1000, 5000, 7500, int(1e5)]
+################################################################################
+
+if __name__ == "__main__":
+
+    # set seed
+    random.seed(0)
+    np.random.seed(0)
+    dp.set_seed(0)
```

Comment:
can we later only need dp.seed(), and inside we set seed for np.random.seed and random.seed

Reply:
This would take a larger update to the code as we would not want to set the seed for numpy or random without the user knowledge. We would have to instead control our own generators and have all usages of random utilize them as well. Feasible, just would take some refactoring. Definitely not opposed.